experimenter/experimenter/experiments/tests/api/v8/test_serializers.py

```diff
@@ -21,6 +24,19 @@
 class TestNimbusExperimentSerializer(TestCase):
     maxDiff = None
 
+    @classmethod
+    def _validate_experiment_schema(
+        cls,
+        application: NimbusExperiment.Application,
+        experiment_data: dict[str, Any],
+    ):
+        if NimbusExperiment.Application.is_sdk(application):
+            schema = SdkNimbusExperiment
+        else:
+            schema = DesktopAllVersionsNimbusExperiment
+
+        schema.model_validate(experiment_data)
```

Comment:
V8 exists specifically so that its schema can diverge from the client schema, so we shouldn't be testing against it here 🙏